app/routes/overview/components/Overview.js

```diff
@@ -19,11 +19,15 @@ import Icon from 'app/components/Icon';
 import truncateString from 'app/utils/truncateString';
 import { Link } from 'react-router';
 import NextEvent from './NextEvent';
+import Poll from 'app/components/Poll';
+import type { PollEntity } from 'app/reducers/polls';
 
 type Props = {
   frontpage: Array<Object>,
   readmes: Array<Object>,
-  loadingFrontpage: boolean
+  loadingFrontpage: boolean,
+  poll: PollEntity,
```

Comment:
This can be `undefined` becuase of the `selectPinnedPolls(state)[0]`.

The type should therefore be `poll: ?PollEntity`.